src/redis/Redis.cpp

```diff
@@ -11,6 +11,11 @@
 
 namespace faabric::redis {
 
+UniqueRedisReply wrapReply(redisReply* r)
```

Comment:
Looks like we have to do a cast to `redisReply` every time we call this, so making the argument whatever `redisCommand` returns and moving the `redisReply` cast into here here would save a few characters on each call.

To make it really succinct, you might me able to move the call to `redisCommand` into here too and use varargs to forward the arguments from this function. Could call it something like `safeRedisCommand` instead.

Reply:
I added a `safeRedisCommand` as suggested, `wrapReply` is still used in a couple of places where it wraps things not directly coming out of `redisCommand` calls